src/ApiRequestPanel.js

```diff
@@ -480,7 +535,6 @@ export class ApiRequestPanel extends AmfHelperMixin(EventsTargetMixin(HeadersPar
     window.removeEventListener('api-response', this._apiResponseHandler);
     this.removeEventListener('api-request', this._apiRequestHandler);
     this.removeEventListener('api-server-changed', this._serverChangeHandler);
-    this.removeEventListener('api-navigation-selection-changed', this._handleNavigationChange);
```

Comment:
Why remove the detaching of the listener?

Reply:
Good catch, this was not intended